### PR TITLE
Use `/bin/bash` instead of `/bin/sh` for execute

### PR DESCRIPTION
**Pull request by lukaszachy**
Fixes just 'local' provision, those using ssh (connect, virtual..) are more difficult and will be fixed later.

Filled #1474 to not forget.

### CODE REVIEW

**Comment by lukaszachy** [comment]
Good, we got further then before and now it isn't tmt's problem  - https://github.com/mkoncek/javapackages-validator/runs/7408633433?check_suite_focus=true

**Comment by lukaszachy** [comment]
Moving to 1.17 - I don't want to break ssh this close to the release

**Comment by lukaszachy** [comment]
@mizdebsk I saw you are using this branch in your config. I'll leave this branch until 1.17 is released and you can revert your changes.

**Comment by lukaszachy** [comment]
@psss @thrix @FrNecas Pleas keep the branch when merging.

**Comment by psss** [comment]
> @psss @thrix @FrNecas Pleas keep the branch when merging.

Is it possible to deactive the auto-branch-delete-upon-merge feature for a specific pull request?

**Review by psss — APPROVED** [review]
Looks good, thanks for improving this. Two nitpicks added in ac4410e.

File: tmt/utils.py [hunk 1]
```diff
@@ -554,11 +554,15 @@ def _run(self,
             environment = None
         self.debug('environment', pprint.pformat(environment), level=4)
 
+        # Set only for shell=True as it would affect command
+        executable = "/bin/bash" if shell else None
```
Comment (happz) [inline tmt/utils.py:558]: I'd suggest moving `/bin/bash` to a module-level "constant", `DEFAULT_SHELL_EXECUTABLE` maybe? "No magic numbers in code" should apply to "magic" strings as well.

Comment (psss) [reply tmt/utils.py:558]: Sounds reasonable, will update that.

**Comment by psss** [comment]
Rebased & squashed.

**Comment by lukaszachy** [comment]
@mizdebsk If you could update your configuration to point to either 'main'  branch or '1.17.0' tag it would be great.
Once you have done that please comment here so I can delete this branch.

**Comment by mizdebsk** [comment]
@lukaszachy I've updated our configuration so that your branch is no longer needed.

**Pull request merged**